src/metabase/db/custom_migrations.clj

```diff
@@ -173,7 +173,7 @@
                      :set    {:json_unfolding true}
                      :where  [:in :metabase_field.id field-ids-to-update]}))))
 
-(defn- update-legacy-field-refs [viz-settings]
```

Comment:
I renamed this function used in the previous migration to differentiate it from `update-legacy-field-refs-in-result-metadata`.